src/app/pages/AccountDetailsPage/index.tsx

```diff
@@ -19,8 +18,6 @@ export const AccountDetailsPage: FC = () => {
   const address = useLoaderData() as string
   const { account, isLoading } = useAccount(scope, address)
 
-  const rosePriceQuery = useGetRosePrice()
```

Comment:
@lukaw3d says:

> hm, keeping requests in Page components is becoming a bit annoying. I would still stick to it tho. Makes tests, storybooks, and reusing components clearer

Reply:
Bit isn't the ticker price like a textbook example of data that should be passed down in context, instead of trickling it through like 3 layers of components? Especially since the price of ROSE is totally independent of the page we are at..

The tests, storybooks, etc is indeed clearer, but then again, those can also be handled by adding the right component wrapper / mock hook...

Reply:
Anyway, I have put it back to page level now.